rclcpp_examples/CMakeLists.txt

```diff
@@ -103,8 +103,18 @@ if(AMENT_ENABLE_TESTING)
 
   # Add each test case.  Multi-executable tests can be specified in
   # semicolon-separated strings, like  exe1;exe2.
-  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters "talker;listener")
-
+  set(example_executables
+    list_parameters_async
+    list_parameters
+    parameter_events_async
+    parameter_events
+    set_and_get_parameters_async
+    set_and_get_parameters
+    "talker\;listener"
+    "add_two_ints_client\;add_two_ints_server"
+    "add_two_ints_client_async\;add_two_ints_server")
```

Comment:
I've replaced the code with this, but I can't get it to work:

```
  # Add each test case.  Multi-executable tests can be specified in
  # semicolon-separated strings, like  exe1;exe2.
  set(example_executables
    list_parameters_async
    list_parameters
    parameter_events_async
    parameter_events
    set_and_get_parameters_async
    set_and_get_parameters
    "talker:listener"
    "add_two_ints_client:add_two_ints_server"
    "add_two_ints_client_async:add_two_ints_server")

  foreach(example_executable_raw ${example_executables})
    set(REPLACE ";" ":" "${example_executable_raw}" example_executable)
    set(RCLCPP_EXAMPLES_EXPECTED_OUTPUT "")
    foreach(executable ${example_executable})
```

what'd be the correct approach?